erigon-lib/kv/mdbx/kv_mdbx.go

```diff
@@ -812,6 +818,13 @@ func (db *MdbxKV) Update(ctx context.Context, f func(tx kv.RwTx) error) (err err
 	return nil
 }
 
+func (db *MdbxKV) StopFlushing() {
+	if db.periodicFlushTicker != nil {
+		db.periodicFlushTicker.Stop() // Stop the ticker
+		db.quitFlushingChan <- struct{}{}
```

Comment:
All .Close methods in Erugon must be safe to call multiple times ( from same goroutine).